From 64e265981b78eaa88727c9bed33a550348fd9b4b Mon Sep 17 00:00:00 2001
From: Federico Cabiddu <federico.cabiddu@gmail.com>
Date: Sat, 9 May 2015 18:16:28 +0200
Subject: [PATCH] modules/nathelper: don't include enclosing bracket in contact
 uri in set_contact_alias

(cherry picked from commit b525ada2d12bf85c385a5f97859afd5d52405354)
---
 modules/nathelper/nathelper.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/nathelper/nathelper.c b/modules/nathelper/nathelper.c
index 8d8c897..efa29c0 100644
--- a/modules/nathelper/nathelper.c
+++ b/modules/nathelper/nathelper.c
@@ -929,8 +929,8 @@ set_contact_alias_f(struct sip_msg* msg, char* str1, char* str2)
 		pkg_free(buf);
 		return -1;
 	}
-	c->uri.s = buf;
-	c->uri.len = len;
+	c->uri.s = buf + br;
+	c->uri.len = len -2*br;
 
 	return 1;
 }
-- 
2.1.4

